plugins/carlabase/CMakeLists.txt

```diff
@@ -1,9 +1,19 @@
+# For MacOS, use old RPATH install_name behavior
+IF (CMAKE_MAJOR_VERSION GREATER 2 AND CMAKE_MINOR_VERSION GREATER 8)
```

Comment:
[`CMP0068`](https://cmake.org/cmake/help/latest/policy/CMP0068.html) is for `CMake >= 3.9`.
Also, you shouldn't compare the CMake version in this way. Suppose one is using CMake 4.0 in the future. Then the check will fail. You can do it like this:
```CMake
IF(NOT CMAKE_VERSION VERSION_LESS 3.9)
# IF(CMAKE_VERSION VERSION_GREATER_EQUAL 3.9) # Available in CMake >= 3.7
        CMAKE_POLICY(SET CMP0068 OLD)
ENDIF()
```